include/deal.II/non_matching/mapping_info.h

```diff
@@ -523,6 +546,8 @@ namespace NonMatching
      */
     AlignedVector<Point<dim, VectorizedArrayType>> unit_points;
 
+    AlignedVector<Point<dim>> unit_points_scalar;
+
```

Comment:
Apart from the fact that I do not like to store the same information of a global property twice (`unit_points` and `unit_points_scalar` refer to the same data, just shuffled around differently), this is (a) undocumented and (b) not a very intuitive name.